README.md

```diff
@@ -966,6 +966,10 @@ There are three styles of `MacosDatePickers`:
   calendar-like interface to select a date.
 * `combined`: provides both `textual` and `graphical` interfaces.
 
+Localization of the time picker is supported by the `weekdaysAbbreviations` and `monthsAbbreviations` parameters (instead of e.g. standard `localizations.narrowWeekdays()` in order to match Apple's spec).
+* `weekdaysAbbreviations` should be a list of 7 strings, one for each day of the week, starting with Sunday
+* `monthsAbbreviations` should be a list of 12 strings, one for each month of the year, starting with January
+
```

Comment:
```suggestion
Localization of the time picker is supported by the `weekdayAbbreviations` and `monthAbbreviations` parameters (instead of e.g. standard `localizations.narrowWeekdays()` in order to match Apple's spec).
* `weekdayAbbreviations` should be a list of 7 strings, one for each day of the week, starting with Sunday
* `monthAbbreviations` should be a list of 12 strings, one for each month of the year, starting with January

```